src/wallet/wallet.h

```diff
@@ -1126,6 +1127,13 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
                           int minDepth=1,
                           bool ignoreSpent=true,
                           bool ignoreUnspendable=true);
+
+    /* Find notes filtered by payment addresses, min depth, ability to spend */
+    void GetFilteredNotes(std::vector<CNotePlaintextEntry>& outEntries,
```

Comment:
Related: See #2913 where I added a Get.. method for z_listunspent.  Looked to refactor, but it needed a maxDepth parameter and also a way to pass back the number of confirmations, which doesn't really belong to struct CNotePlaintextEntry.